serenata_toolbox/chamber_of_deputies/dataset.py

```diff
@@ -137,3 +132,6 @@ def __translate_file(self, csv_path):
                     encoding='utf-8')
 
         return output_file_path
+
+    def __parse_float(self, string):
```

Comment:
Usually in Python use kind of prevention is not necessary.

> “Private” instance variables that cannot be accessed except from inside an object don’t exist in Python. However, there is a convention that is followed by most Python code: a name prefixed with an underscore (e.g. _spam) should be treated as a non-public part of the API (whether it is a function, a method or a data member). It should be considered an implementation detail and subject to change without notice.

source: https://docs.python.org/3/tutorial/classes.html#private-variables

@Irio, do you feel that it still worth it?